plugins/content_distribution/providers/tvinci/lib/feed/TvinciDistributionFeedHelper.php

```diff
@@ -137,24 +165,29 @@ private function createXml($action)
 		$this->_doc = new DOMDocument();
```

Comment:
I hoped to get rid of all this XML construction by transforming the kMRSS to the WSDL XML using a single XSL, can we do that?